stdlib/SparseArrays/test/sparse.jl

```diff
@@ -318,8 +318,7 @@ end
         a = sprand(10, 5, 0.7)
         b = sprand(5, 15, 0.3)
         @test maximum(abs.(a*b - Array(a)*Array(b))) < 100*eps()
-        @test maximum(abs.(SparseArrays.spmatmul(a,b,sortindices=:sortcols) - Array(a)*Array(b))) < 100*eps()
-        @test maximum(abs.(SparseArrays.spmatmul(a,b,sortindices=:doubletranspose) - Array(a)*Array(b))) < 100*eps()
+        @test maximum(abs.(SparseArrays.spmatmul(a,b) - Array(a)*Array(b))) < 100*eps()
```

Comment:
This test is redundant with the one above, I believe. We should delete and maybe add another that tests the transpose multiplication case.

Reply:
I will work on that tomorrow. I did not look at the `*` function yet. What do you mean exactly by the transpose case? `A * B'` or `A' * B` or both (if A and B are SparseMatrixCSC).

Reply:
Yes, I believe that `a*b` and `spmatmul(a,b)` are exactly the same. The `*` just calls `spmatmul`, and hence it is the same test:

https://github.com/JuliaLang/julia/blob/855959124dbafadafdcd324771ec06831bc79a1a/stdlib/SparseArrays/src/linalg.jl#L142

But good to test other code paths for `a*b'` and `a'*b` with something like:
```
@test maximum(abs.(a'*b - Array(a')*Array(b))) < 100*eps()
@test maximum(abs.(a*b' - Array(a)*Array(b'))) < 100*eps()
```